### ISSUE TITLE
Blood does not appear to be moving compared to the background

### ISSUE THREAD

**Issue by stefanuros** [ZaneLittle/SundayStroll#46]
Is there some way to programatically move the blood at the speed of background?

**Comment by ZaneLittle** [comment]
This might be a non-issue. When the player is hit, the background should stop moving (relate #49). It would be odd to keep elements moving when the player is dead as it signifies that the player is still in movement.

**Comment by stefanuros** [comment]
This may need to be done later when we have the global config file. The
background would need to know when the player is dead. Making it stop is as
simple as setting speed to 0 but triggering it will need to be looked into.

**Comment by ZaneLittle** [comment]
True, I was going to look into that as part of the work on the game over state

**Comment by stefanuros** [comment]
I think thta should work. I seem to remember a function that finds a game
object based on name. We can locate the game object and set speed to  from
the game over script.
